src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/PurchaseDialog/index.js

```diff
@@ -59,7 +59,7 @@ const PurchaseDialog = ({ classes, show, onClose, startDepositLoader, startWithd
     try {
       const amountInAGI = amount[txnTypes.WITHDRAW];
       const amountInCogs = agiToCogs(amountInAGI);
-      const response = await sdk.account.withdrawFromEscrowAccount(amountInCogs);
+      await sdk.account.withdrawFromEscrowAccount(amountInCogs);
       setAmount({});
```

Comment:
Why do you need to set the amount here?

Reply:
https://github.com/singnet/refactored-dapp/pull/160#discussion_r316110393